homeassistant/components/device_tracker/legacy.py

```diff
@@ -203,6 +203,31 @@ async def async_setup_integration(hass: HomeAssistant, config: ConfigType) -> No
     """Set up the legacy integration."""
     tracker = await get_tracker(hass, config)
 
+    async def async_see_service(call: ServiceCall) -> None:
+        """Service to see a device."""
+        # Temp workaround for iOS, introduced in 0.65
+        data = dict(call.data)
+        data.pop("hostname", None)
+        data.pop("battery_status", None)
+        await tracker.async_see(**data)
+
+    hass.services.async_register(
+        DOMAIN, SERVICE_SEE, async_see_service, SERVICE_SEE_PAYLOAD_SCHEMA
+    )
+
+    # The platforms load in a tracked task to ensure
+    # device tracker setup can continue and config
+    # entry integrations are not waiting for
+    # legacy device tracker platforms to be set up.
+    hass.async_create_task(
+        _async_setup_legacy_integration(hass, config, tracker), eager_start=True
+    )
```

Comment:
This one is a single task where the platforms get setup inside because it still needs to do `restore` after they are done